tests/runtests.sh

```diff
@@ -5,31 +5,18 @@ set -e
 lambdapi='dune exec -- lambdapi check'
 TIMEFORMAT="%Es"
 
-out=/tmp/lambdapi.output
-
 ok_tests() {
     for f in 'tests/OK/a b/escape file.lp' tests/OK/*.lp tests/OK/*.dk
     do
         case $f in
             tests/OK/why3*.lp);; #FIXME
             *)
                 echo lambdapi check $options $f ...
-                $lambdapi "$f" > $out 2>&1
```

Comment:
Oh I understand now why we where doing this before: to print lambdapi output only when there was an error!... We have to revert the changes then. :-) But it was perhaps not completely correct. Shouldn't we instead do:
```
                $lambdapi "$f" > $out 2>&1 || (cat $out; exit 1)
```

Reply:
If I understood well, since we have `set -e` we can just redirect stdout and leave sterr display to terminal if any error happen. `set -e` will stop the script and exit 1.

Reply:
But it is better to print the lambdapi output before the error messages.